shared/test/test_hip_chat.rb

```diff
@@ -0,0 +1,80 @@
+require 'fakeweb'
+require 'minitest/autorun'
+require 'rack/test'
+require_relative '../../deployment'
+require_relative '../../lib/cdo/hip_chat'
+
+class HipchatTest < Minitest::Test
+
+  MAX_WAIT_SECONDS = 10
```

Comment:
MAX_WAIT_SECONDS looks like it's unused.   Unless, it's some kind of Minitest config option I don't know about...